exporter/exporterhelper/internal/obsexporter.go

```diff
@@ -111,7 +114,7 @@ func (or *ObsReport) EndProfilesOp(ctx context.Context, numSpans int, err error)
 // the updated context and the created span.
 func (or *ObsReport) startOp(ctx context.Context, operationSuffix string) context.Context {
 	spanName := or.spanNamePrefix + operationSuffix
-	ctx, _ = or.tracer.Start(ctx, spanName)
+	ctx, _ = or.tracer.Start(ctx, spanName, or.spanAttrs)
```

Comment:
Shouldn't we have a changelog since it changes the internal span attributes?

Reply:
Done.